packages/cosmic-swingset/lib/ag-solo/fake-chain.js

```diff
@@ -58,10 +58,6 @@ export async function connectToFakeChain(basedir, GCI, delay, inbound) {
     noFakeCurrencies: process.env.NO_FAKE_CURRENCIES,
   };
   const stateDBdir = path.join(basedir, `fake-chain-${GCI}-state`);
-  function doOutboundBridge(dstID, _obj) {
```

Comment:
What was formerly improper?

I'm lacking context.

_this is a comment on the commit, not this line in particular_